src/rtpghi~.c

```diff
@@ -1,14 +1,30 @@
-//
-//  test.c
-//
-//
 //  Created by maximilian helligrath on 28.07.23.
 //
 
 #include <m_pd.h>
 #include <phaseret.h>
 #include <ltfat.h>
 
+#if !defined(PD_FLOATSIZE)
+  /* normally, our floats (t_float, t_sample,...) are 32bit */
+# define PD_FLOATSIZE 32
+#endif
+
```

Comment:
`PD_FLOATSIZE` has been defined in Pd-headers since 0.45-0 (released more than 10 years ago).
it's probably safe to assume that it is already defined (and just error out if it is missing).

otoh, there's no harm done.